### PR TITLE
Add new synchronization permissions

### PR DESCRIPTION
**Pull request by ArvindHarinder1**

### CODE REVIEW

File: changelog/Microsoft.DirectoryServices.json [hunk 1]
```diff
@@ -1,5 +1,31 @@
...
+          "ChangeType": "Addition",
+          "Description": "Added the `Synchronization.ReadWrite.All` application and permission to configure synchronization settings.",
+          "Target": "synchronization"
+        },
+        {
+          "Id": "00c006a9-0eaf-423f-9ed1-ge86808382ab",
+          "ApiChange": "Permission",
+          "ChangedApiName": "Synchronization.Read.All",
+          "ChangeType": "Addition",
+          "Description": "Added the `Synchronization.Read.All` application and permission to read synchronization settings.",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.DirectoryServices.json:18]: Suggestion
```suggestion
          "Description": "Added the `Synchronization.ReadWrite.All` application and delegated permissions to configure [synchronization settings](https://learn.microsoft.com/en-us/graph/api/resources/synchronization-overview?view=graph-rest-beta).",
          "Target": "synchronization"
        },
        {
          "Id": "00c006a9-0eaf-423f-9ed1-ge86808382ab",
          "ApiChange": "Permission",
          "ChangedApiName": "Synchronization.Read.All",
          "ChangeType": "Addition",
          "Description": "Added the `Synchronization.Read.All` application and delegated permissions to read [synchronization settings](https://learn.microsoft.com/en-us/graph/api/resources/synchronization-overview?view=graph-rest-beta).",
```

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Hi @ArvindHarinder1 -

In the spirit of encouraging least privilege, we're avoiding documenting _Directory.Read.All_ and _Directory.ReadWrite.All_ if an API doesn't **need** either. If _Synchronization.Read.All_ and _Synchronization.ReadWrite.All_ are enough for the synchronization operations, please remove the Directory.* permissions from the permissions tables. Let me know if we need to document either.

Let one suggestion for the changelog.

**Comment by ArvindHarinder1** [comment]
> Hi @ArvindHarinder1 -
> 
> * In the spirit of encouraging least privilege, we're avoiding documenting _Directory.Read.All_ and _Directory.ReadWrite.All_ if an API doesn't **need** either. If _Synchronization.Read.All_ and _Synchronization.ReadWrite.All_ are enough for the synchronization operations, please remove the Directory.* permissions from the permissions tables. Let me know if we need to document either.
> * Also, document the new permissions in the [permissions reference](https://learn.microsoft.com/en-us/graph/permissions-reference).
> * Left one suggestion for the changelog.

@FaithOmbongi  thanks for the feedback. I've gone ahead and addressed the feedback.

**Comment by FaithOmbongi** [comment]
Selected the re-request review icon.

**Review by FaithOmbongi — APPROVED** [review]
Thank you @ArvindHarinder1 for your work on this PR. Approved.

**Pull request merged**